crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -468,8 +468,9 @@ impl CumulocityConverter {
                     threshold: self.size_threshold.0,
                 });
             }
-        };
```

Comment:
I expected this else case in the `try_convert_te_topics` function inside the `if self.entity_store.get(&source).is_none() {` block, so that it applies to all data types. If the entity is not found and could not be auto-registered either, it shouldn't even get to this function.